Mathlib/Algebra/BigOperators/Ring.lean

```diff
@@ -68,6 +68,16 @@ end Semiring
 
 section Semiring
 
+theorem dvd_sum [NonUnitalSemiring β]
```

Comment:
Since you are at it, do you mind generalize also [List.dvd_sum](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/List/BigOperators/Lemmas.html#List.dvd_sum)?

Reply:
Yes, I'd prefer that we make the generalizations necessary to keep the old one-line proof.